bin/dropsonde

```diff
@@ -69,7 +69,7 @@ class Dropsonde
   desc 'Submit a telemetry report'
   command :submit do |c|
     c.desc 'Telemetry endpoint'
-    c.flag [:endpoint], :default_value => 'https://prod.dujour.k8s.puppet.net'
```

Comment:
This was actually already removed because we've moved Dujour off this Kubernetes infrastructure, but we're working on adding it back temporarily.